src/recipes.ts

```diff
@@ -1,207 +1,4776 @@
-import { Item, items, Machine, Minutes, Ore, Quantity } from "./items"
+/**
+ * recipes.ts
+ * All recipes
+ * lgfrbcsgo & Nikolaus - October 2020
+ */
 
+import { Item, Ore, items, Minutes, Quantity } from "./items"
+
+/**
+ * Batch holds a product and the production batch size
+ */
 export interface Batch {
     readonly item: Item
     readonly quantity: Quantity
 }
 
+/**
+ * Recipe holds all relevant recipe information
+ */
 export interface Recipe {
     readonly product: Batch
+    readonly time: Minutes
+    readonly industry: string
```

Comment:
`industry` should probably be an `Item` to avoid typos.

Reply:
`Item` are things to be produced by the factory. Here, the `industry` is the machine that builds the `Item`. I've made it an enum instead. Done in https://github.com/lgfrbcsgo/du-factory-generator/commit/b2ca5abf70d5f5d1f8272961ae611d9e8b5b9bf3

Reply:
Having the actual `Item` here would avoid redundancy. If the name of the `Assembly line L ` changes, we only need to change it in one place. It could be also beneficial for the UI. In case you e.g. don't own a glass furnace, you could directly generate a factory to build a glass furnace.

Reply:
I leave this up to you to decide.

Reply:
```suggestion
    readonly industry: Industry
```

Reply:
I agree now. Changed in https://github.com/lgfrbcsgo/du-factory-generator/commit/c491d668819652b515c0ab3a09f4b76020f30287